manifests/params.pp

```diff
@@ -60,4 +82,19 @@
       fail("Unsupported operating system: ${facts['os']['name']}")
     }
   }
+
+  case $version_corosync {
```

Comment:
can you rewrite this part to use the selector pattern? https://puppet.com/docs/puppet/5.5/lang_conditional.html#selectors